cmd/libs/go2idl/go-to-protobuf/package.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"k8s.io/kubernetes/cmd/libs/go2idl/generator"
+	"k8s.io/kubernetes/cmd/libs/go2idl/types"
+)
+
+func newProtobufPackage(packagePath, packageName string) *protobufPackage {
+	pkg := &protobufPackage{
+		PackageName: packageName,
+		PackagePath: packagePath,
```

Comment:
I see. I don't feel too strongly about the proto files. We need to be careful with the generated functions, like maybe deleting generated files before running the generator. We don't want to get into today's situation where we can't run the generator because it doesn't compile.